integration-cli/docker_cli_build_test.go

```diff
@@ -7371,7 +7369,13 @@ WORKDIR /foo/bar
 	c.Assert(err, checker.IsNil, check.Commentf("Output: %s", out))
 
 	out, _ = dockerCmd(c, "inspect", "--format", "{{ json .Config.Cmd }}", image)
-	c.Assert(strings.TrimSpace(out), checker.Equals, `["sh"]`)
+
+	// The Windows busybox image has a blank `cmd`
```

Comment:
Completely orthogonal.... the issue is that the cmd is overwritten, not what the default value is to start with. If you revert 29679, you can see that the test is valid as CMD is being overwritten incorrectly by the WORKDIR statement as shown below:

```
----------------------------------------------------------------------
FAIL: docker_cli_build_test.go:7318: DockerSuite.TestBuildWorkdirImageCmd

docker_cli_build_test.go:7334:
    c.Assert(strings.TrimSpace(out), checker.Equals, lookingFor)
... obtained string = "[\"cmd\",\"/S\",\"/C\",\"#(nop) WORKDIR C:\\\\foo\\\\bar\"]"
... expected string = "null"

OOPS: 0 passed, 1 FAILED
--- FAIL: Test (2.29s)
FAIL
exit status 1
FAIL    github.com/docker/docker/integration-cli        2.492s
```